clippy_lints/src/else_if_without_else.rs

```diff
@@ -0,0 +1,71 @@
+//! lint on if expressions with an else if, but without a final else branch
+
+use rustc::lint::*;
+use syntax::ast::*;
+
+use utils::{in_external_macro, span_lint_and_sugg};
+
+/// **What it does:** Checks for usage of if expressions with an else if branch,
+/// but without a final `else` branch.
+///
+/// **Why is this bad?** Some coding guidelines require this.
```

Comment:
Can you reference the MISRA standard rule here?

Reply:
Yep!